content/sdk/10.js/00.ethers/10.v5/03.providers.md

```diff
@@ -1,38 +1,35 @@
 ---
 title: Providers
-description:
+description: A guide to Web3 Provider objects for ZKsync integration
+tags: ["zksync", "ethereum", "zksync Era", "transactions", "provider"]
 ---
 
 A Web3 Provider object provides application-layer access to underlying blockchain networks.
 
-The [`zksync-ethers`](https://www.npmjs.com/package/zksync-ethers/v/5.0.0) library supports
-provider methods from the [`ethers.js`](https://docs.ethers.io/v5/api/providers) library and
-supplies additional functionality.
+The [`zksync-ethers`](https://www.npmjs.com/package/zksync-ethers/v/5.0.0) library supports provider methods from
+the [`ethers.js`](https://docs.ethers.io/v5/api/providers) library and supplies additional functionality.
 
-Two providers are available:
+## Available providers
 
-- [`Provider`](#provider): Supplies the same functionality as
-  [`ethers.providers.JsonRpcProvider`](https://docs.ethers.org/v5/api/providers/jsonrpc-provider/#JsonRpcProvider)
-  and extends it with ZKsync-specific methods.
-- [`Web3Provider`](#web3provider): Extends the ZKsync Era [`Provider`](#provider) class to make it more compatible
-  with Web3 wallets.
+- **Provider**: Extends [`ethers.JsonRpcProvider`](https://docs.ethers.org/v5/api/providers/jsonrpc/#about-jsonrpcProvider)
```

Comment:
It looks like a variety of Ethers docs links are broken, please go through and check that they're up to date.

```suggestion
- **Provider**: Extends [`ethers.JsonRpcProvider`](https://docs.ethers.org/v5/api/providers/jsonrpc-provider/)
```